v0/sources/shopify/transform.js

```diff
@@ -48,11 +79,36 @@ const trackPayloadBuilder = (event, shopifyTopic) => {
   message.setEventType(EventType.TRACK);
   message.setEventName(shopifyTopic);
   Object.keys(event)
-    .filter(key => !key.includes(["type", "event"]))
+    .filter(
+      key =>
+        ![
+          "type",
+          "event",
+          "line_items",
+          "customer",
+          "shipping_address",
+          "billing_address"
+        ].includes(key)
+    )
     .forEach(key => {
       message.setProperty(`properties.${key}`, event[key]);
     });
-  if (event.user_id) {
+  const { line_items: lineItems } = event;
+  const productsList = getProductsListFromLineItems(lineItems);
+  message.setProperty("properties.products", productsList);
+  if (event.customer) {
+    message.setPropertiesV2(
+      event.customer,
+      MAPPING_CATEGORIES[EventType.IDENTIFY]
+    );
+  }
+  if (event.shipping_address) {
+    message.setProperty("traits.shippingAddress", event.shipping_address);
+  }
+  if (event.shipping_address) {
```

Comment:
I believe this should've been `event.billing_address`.

Reply:
Fixed